nodeup/pkg/model/kubelet.go

```diff
@@ -175,6 +175,15 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 		kubeletConfig.BootstrapKubeconfig = ""
 	}
 
+	if kubeletConfig.ExperimentalAllowedUnsafeSysctls != nil {
+		// The ExperimentalAllowedUnsafeSysctls flag was renamed in k/k #63717
+		if b.IsKubernetesGTE("1.11") {
+			glog.V(1).Info("ExperimentalAllowedUnsafeSysctls was renamed in k8s 1.11+, please use AllowedUnsafeSysctls instead.")
```

Comment:
I like this - it feels like mapping it is the correct thing to do here.

But - it's in nodeup, and most people don't see that.

I'm wondering if we should add this to the configuration generation in kubeup itself.  (It's also better to have less logic in nodeup generally - nodeup is ideally dumb, though in practice there are lots of places where it can't be).

We could have a method which does these flags removed / renamed checks, and remaps / warns / errors as appropriate.  In this case remapping seems like the right call, but I think we should actively warn the user, to encourage them to change their configuration